dev-requirements.txt

```diff
@@ -16,7 +16,7 @@ codacy-coverage
 Sphinx==1.6.5
 sphinx_rtd_theme
 flasgger==0.9.1
-PyYAML==4.2b1
```

Comment:
(trivial) as Pauline mentioned in a different PR that I cannot remember which, maybe it is okay to remove this duplicate requirement from `dev-requirements.txt` if it already exists in `requirements.txt`?

Reply:
O yes indeed thank you 🤦‍♂️